tests/impl.cpp

```diff
@@ -6968,7 +6968,14 @@ result_t test_mm_extract_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_extract_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    int64_t *_a = (int64_t *) impl.mTestIntPointer1;
+    const int imm = 1;
+
+    __m128i a = do_mm_load_ps((const int32_t *) _a);
+    __int64_t c = _mm_extract_epi64(a, imm);
+
+    ASSERT_RETURN(c == *(_a + (imm & 1)));
+    return TEST_SUCCESS;
```

Comment:
Could you rewrite the test case using the switch case?
https://github.com/DLTcollab/sse2neon/blob/f3f567484d099f74aaa51880db46b1434c960638/tests/impl.cpp#L4803